packages/mermaid/src/docs/config/theming.md

```diff
@@ -133,7 +133,7 @@ The theming engine will only recognize hex colors and not color names. So, the v
 | -------------------- | ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
 | darkMode             | false                              | Affects how derived colors are calculated. Set value to `true` for dark mode.                                                    |
 | background           | #f4f4f4                            | Used to calculate color for items that should either be background colored or contrasting to the background                      |
-| fontFamily           | trebuchet ms, verdana, arial       |                                                                                                                                  |
+| fontFamily           | trebuchet ms, verdana, arial       | Font family for diagram text; can be customized via `themeVariables`                                                             |
```

Comment:
This table is already part of theme variables, so we don't need to mention "can be customized via `themeVariables`" here.